.github/workflows/deploy.yml

```
@@ -78,7 +93,7 @@ jobs:
       - name: Checkout repo 🛎
         uses: actions/checkout@v4
         with:
-          ref: "${{ env.BRANCH_NAME }}"
+          ref: "669_insertUI@dev"
```

```suggestion
          ref: "${{ env.BRANCH_NAME }}"
```

Should revert the branch checkout before merging. This was changed just to test the deployment.